### ISSUE TITLE
EL7: add/configure required service if iptables should be used

### ISSUE THREAD

**Issue by jouvin** [quattor/template-library-os#83]
In EL7, `iptables` services is not installed by default as it is replaced by `firewalld`. But it is still possible to install the `iptables` service with the following steps:

```
systemctl stop firewalld
systemctl mask firewalld
yum install iptables-services
systemctl enable iptables
systemctl enable ip6tables
```
The template library, should have a variable to select between `iptables` and `firewalld` and do what is appropriate to use `iptables`.

It can be desirable until `firewalld` is properly supported (https://github.com/quattor/configuration-modules-core/issues/961).

I labelled this issue as a bug because as explained in https://github.com/quattor/configuration-modules-core/issues/961 without this configuration `ncm-iptables` succeeds (given the impression that the system is protected) when it should fail.

**Comment by jrha** [comment]
The real question is, is it enough of a bug to delay the release any further?

**Comment by jrha** [comment]
Fixed by #84?

**Comment by jouvin** [comment]
Yes it is. the main reason I didn't state it in #84 was the remaining issue in `ncm-systemd` but it has been fixed by https://github.com/quattor/configuration-modules-core/pull/965.
